data/helpers.d/network

```diff
@@ -0,0 +1,15 @@
+# Check the path syntax
+# Check the slash at the beginning of path and its absence at ending
+#
+# example: ynh_check_path
+#
+# usage: ynh_check_path
+ynh_check_path () {
+	test -n "$path" || ynh_die "The path variable is empty."
+	if [ "${path:0:1}" != "/" ]; then    # If the first character is a /
```

Comment:
There is an error in the comment, it should be:
`# If the first character is not a /`